recipes/sqlite_zstd_vfs/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+cmake $CMAKE_ARGS \
+    -DCMAKE_BUILD_TYPE=Release \
+    -B build \
+
+
+ls -l /Library/Frameworks/Mono.framework/Versions/Current/include || true
+
+
+sed -i.bak '1i\
```

Comment:
If you use `sed`, add it please to `build` to ensure that you get the same `sed` interface on all platforms.

Reply:
Thanks, added in [6c997ea](https://github.com/conda-forge/staged-recipes/pull/28838/commits/6c997ea4bbe8ac842a4668a47ea66cceeca04550)